### PR TITLE
Mentor/volunteer

### PR DESCRIPTION
**Pull request by jfeng5529**
## Description

Updated mentor/volunteer dashboards

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
  - [ ] I wrote a design doc: *Replace this with a link to your design doc's short link*
  - [ ] I got input from a developer, specifically from: *Replace with the names of who gave advice*
  - [ ] I wrote a migration guide: *Replace with a link to your migration guide*

[issue database]: https://github.com/bostonhacks/cacioepepe/issues
[Contributor Guide]: https://github.com/bostonhacks/cacioepepe/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: src/components/volunteer/VolunteerApp.vue [hunk 1]
```diff
@@ -238,14 +238,28 @@
...
+                    =======
                     <a
                       target="_blank"
                       href="https://static.mlh.io/docs/mlh-code-of-conduct.pdf"
                       @click.stop
                     >
                       MLH code of conduct
                     </a>
-                    ？
+                    >>>>>>> 45fe38623f4bed107c9ff070b4627afb1a457d19 ？
```
Comment (rishabnayak) [inline src/components/volunteer/VolunteerApp.vue:262]: merge conflicts left in, please fix

**Review by rishabnayak — CHANGES_REQUESTED** [review]

File: src/components/volunteer/VolunteerApp.vue (hunk 1 above)
Comment (jfeng5529) [reply src/components/volunteer/VolunteerApp.vue]: fixed

**Comment by rishabnayak** [comment]
@jfeng5529 does the slack component work as expected?

**Comment by jfeng5529** [comment]
> @jfeng5529 does the slack component work as expected?

I have not got around to fix it yet, but right now it is just a list of the channels they can join - the one they have in the hacker dashboard

**Pull request merged**